### ISSUE TITLE
Training related parameters question

### ISSUE THREAD

**Issue by Zenobia7** [webtoon/matteformer#1]
Hello, author. First of all, thank you very much for providing such a good code. I would like to ask, is the number of background pictures you used in training 22W of all the pictures in coco data set? Or only part of it? Then, the initial learning rate used in the paper is 1E-4, and the initial learning rate set in the code is 1E-3. Would you like to confirm which one is more appropriate?

**Comment by pgt4861** [comment]
Hi, Thank you for your interest.
We used all coco2014 training images as backgrounds and randomly sampled them when training.
Following the default training configurations in the code is recommended.

**Comment by Zenobia7** [comment]
> Following the default training configurations in the code is recommended.

Thank you very much for your reply. I still have some problems with the setting of learning rate.When my iters exceeds warm up step, loss will not be able to converge. I tried two strategies (1) Step decrease directly from the initial learning rate, 2) gradually increase and then warm up step decrease gradually), such problems will occur. May I ask whether such problems will occur in your side?
_(1)Step decrease directly from the initial learning rate:_
![46dd97768a00185855a3ff4791468c5](https://user-images.githubusercontent.com/28126038/167327373-104dff80-350e-490f-9bd8-7d64f71a8098.png)
_(2) gradually increase and then warm up step decrease gradually:_
![image](https://user-images.githubusercontent.com/28126038/167327513-ee1a8124-6957-4be8-a71a-48bf5c552541.png)

**Comment by mexicantexan** [comment]
I also had this issue :/
